packages/extension-redis/src/Redis.ts

```diff
@@ -364,5 +364,6 @@ export class Redis implements Extension {
     await this.redlock.quit()
     this.pub.disconnect(false)
     this.sub.disconnect(false)
+    this.sub.off('messageBuffer', this.handleIncomingMessage)
```

Comment:
this probably won't help at all, as onDestroy is called when the server shuts down.